daml.yaml

```diff
@@ -4,7 +4,7 @@
 sdk-version: 2.6.0
 name: daml-finance
 source: src/test/daml
-version: 1.2.14.99.20230615.1
+version: 1.2.14.99.20230615.2
```

Comment:
Just fyi, the code for the new tutorial will probably not be included in the built assembly. You should update `docs/scripts/build-assembly.sh`